docs/user-guide.rst

```diff
@@ -30,7 +30,7 @@ so that you don't have to:
 
     http = urllib3.PoolManager()
 
-To make a request use :meth:`~poolmanager.PoolManager.request`:
+To make a request use :meth:`~request.RequestMethods.request`:
```

Comment:
```suggestion
To make a request use :meth:`~urllib3.PoolManager.request`:
```

This only changes the warning, but you still have a warning because this does not link to anywhere in the documentation. The fix in this case is to add `:inherited-members:` to `urllib3.PoolManager` in `docs/reference/urllib3.poolmanager.rst`. And since it's documented as `urllib3.PoolManager`, I've found that using that in the text allows Sphinx to find it, hence the suggestion.